### DISCUSSION TITLE
Using pinit package to annotate equations

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3581] [Q&A] [answered]
I am using the following code, whose output is below and is based on [pinit](https://github.com/OrangeX4/typst-pinit). However, the output I need has the arrows aligned very differently. Could someone please help me? Thanks!

```
#import "@preview/pinit:0.1.3": *

$
#pin(1)Y #pin(2) =
beta_0 + beta_1 #pin(3)X_1 #pin(4) + ... + beta_k #pin(5) X_k #pin(6) + epsilon
$

#pinit-highlight(1, 2)
#pinit-highlight(3, 4)
#pinit-highlight(5, 6)

#pinit-point-from(2)[Known]
#pinit-point-from(4)[]
#pinit-point-from(6)[]

```

Output from the code:

![image](https://github.com/typst/typst/assets/75945439/a0b1e551-f14a-430e-b103-c7c6fd256d96)

What I need (please, notice that the arrow tips are wanted):

![image](https://github.com/typst/typst/assets/75945439/c1ccd763-8b75-487b-acf3-271d9ca5b3eb)

**Comment by AxiomOfChoices** [comment, marked as answer]
How about this?

```
#import "@preview/pinit:0.1.3": *

$
#pin(1)Y#pin(2) =
beta_0 + beta_1 #pin(3)X_1#pin(4) + ... + beta_k #pin(5)p_k#pin(6) + epsilon
$
#pinit-highlight(1, 2)
#pinit-highlight(3, 4)
#pinit-highlight(5, 6)

#pinit-point-from(1)[#pin(7)Known]
#pinit-arrow(7,3,start-dx: 15pt, start-dy: -4pt, end-dx: 5pt, end-dy: 5pt)
#pinit-arrow(7,5,start-dx: 45pt, start-dy: -4pt, end-dx: 5pt, end-dy: 5pt)
```

![image](https://github.com/typst/typst/assets/14252368/74f9dc66-838c-4f73-ba4f-d14faf107bfd)

**Comment by psads-git** [comment]
That is perfect! Thank you a lot, @AxiomOfChoices!
